### ISSUE TITLE
Total request time adds up with HTTP 404 responses

### ISSUE THREAD

**Issue by fdemmer** [dobarkod/django-queryinspect#22]
Using this with Django 1.9.12 and just observed, that the time reported in the stats continues to grow, with repeated 404 responses (via `raise Http404`). So after a while i get:

```
[SQL] 1 queries (0 duplicates), 0 ms SQL time, 175554 ms total request time
```

After a response with HTTP 200, the timer seems to reset.

**Comment by senko** [comment]
Hey @fdemmer I believe you fixed this with your commit https://github.com/dobarkod/django-queryinspect/commit/e771f3803f156b67693f578eddef7e0523d2c8ff (at least can't reproduce it any more). 👍

**Comment by senko** [comment]
Closing as fixed in #28.
